packages/tools/client-debugger/client-debugger/src/messaging/DebuggerMessages.ts

```diff
@@ -66,4 +87,32 @@ export interface ContainerStateChangeMessage
 	type: "CONTAINER_STATE_CHANGE";
 }
 
+/**
+ * Outbound event indicating Container connected.
```

Comment:
From the perspective of this library, these 3 messages are `inbound` rather than `outbound`. I.e. this library listens for those message events, but it doesn't emit them.

Reply:
Make sense! Update the comment here.